server/api/controllers/utils.py

```
@@ -0,0 +1,41 @@
+from dotenv import load_dotenv
+
+load_dotenv()
+JWT_SECRET = os.getenv("JWT_SECRET", "secret123")
+
+
+def get_user_id_from_token() -> int:
+    """Extract user ID from JWT token in Authorization header"""
+    auth_header = request.headers.get("Authorization")
+    
+    if not auth_header or not auth_header.startswith("Bearer "):
+        raise ValueError("Missing or invalid Authorization header")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)